notes-service/src/main/java/org/exoplatform/wiki/listener/AnalyticsAddTagListener.java

```diff
@@ -0,0 +1,49 @@
+package org.exoplatform.wiki.listener;
```

Comment:
For new classes in Meeds, please use package name `io.meeds` instead of `org.exoplatform`
(To apply on all classes please to not repeat the comment multiple times)

```suggestion
package io.meeds.wiki.listener;
```

Reply:
done